pkg/project/resource_project.go

```diff
@@ -217,9 +217,14 @@ func projectResource() *schema.Resource {
 		},
 
 		"repos": {
-			Type:        schema.TypeSet,
-			Optional:    true,
-			Elem:        &schema.Schema{Type: schema.TypeString},
+			Type:     schema.TypeSet,
+			Optional: true,
+			Elem: &schema.Schema{
+				Type: schema.TypeString,
+				//ValidateDiagFunc: projectKeyValidator,
+			},
+			MinItems:    0,
+			MaxItems:    50,
```

Comment:
Because this is a semi artificial limitation, the documentation should say as such and why.
I think an override is also in order. Let's you me and Alex have a chat today